src/schema/rules/sidecars/events.yaml

```
@@ -12,3 +12,14 @@ StimulusPresentation:
   fields:
     StimulusPresentation: recommended
     VisionCorrection: optional
+
+StimulusPresentation_for_eyetracking:
+  selectors:
+    - suffix == "events"
+    - intersects([suffix], ["eyetrack"])
+  fields:
+    StimulusPresentation.ScreenDistance: required
+    StimulusPresentation.ScreenOrigin: required
+    StimulusPresentation.ScreenRefreshRate: recommended
+    StimulusPresentation.ScreenResolution: required
+    StimulusPresentation.ScreenSize: required
```

For some reason I can't see a comment where I would swear I followed up on this and asked @Remi-Gau  to finalize this :eyes:

I also tagged @effigies, @tsalo and @rwblair for support about the schema.

I think I was proposing something of the sort of:

```suggestion
StimulusPresentation_for_eyetracking:
  selectors:
    - suffix == "events"
    - (suffix == "physio").PhysioType == "eyetrack"
  fields:
    StimulusPresentation.ScreenDistance: required
    StimulusPresentation.ScreenOrigin: required
    StimulusPresentation.ScreenRefreshRate: recommended
    StimulusPresentation.ScreenResolution: required
    StimulusPresentation.ScreenSize: required
```

The idea is that, for those events files matching tasks with physio files AND those physio files have the metadata PhysioType=eyetrack, then make these StimulusPresentation fields REQUIRED.

---

Moved to bids-standard/bids-validator#143

---

Added in https://github.com/bids-standard/bids-specification/pull/1128/commits/abe6036d364b53fca313d750f4234dc71e805762